client/keys/parse.go

```diff
@@ -0,0 +1,72 @@
+package keys
+
+import (
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/tendermint/tendermint/libs/bech32"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+var bech32Prefixes = []string{
+	sdk.Bech32PrefixAccAddr,
+	sdk.Bech32PrefixAccPub,
+	sdk.Bech32PrefixValAddr,
+	sdk.Bech32PrefixValPub,
+	sdk.Bech32PrefixConsAddr,
+	sdk.Bech32PrefixConsPub,
+}
+
+func parseKeyStringCommand() *cobra.Command {
+	return &cobra.Command{
+		Use:   "parse <hex-or-bech32-address>",
+		Short: "Parse address from hex to bech32 and vice versa",
+		Long: `Convert and print to stdout key addresses and fingerprints from
+hexadecimal into bech32 cosmos prefixed format and vice versa.
+`,
+		Args: cobra.ExactArgs(1),
+		RunE: parseKey,
```

Comment:
Not totalllyyyy necessary, but it would be cool if this respected the `-o json` flag. We'd only need two private/internal types here with `String()` methods.

Reply:
Done in 3715a01f264af5c9c9a272afdd45c3aefeb0e803
Please check